plugins/GSdx/GSDevice11.cpp

```diff
@@ -434,7 +434,12 @@ void GSDevice11::SetExclusive(bool isExcl)
 
 void GSDevice11::Flip()
 {
-	m_swapchain->Present(m_vsync, 0);
+	// PCSX2 sends a negative value (-1) to GSdx when adaptive mode is specified for Vsync
+	// This mode is exclusive to OpenGL at the moment and passing a negative value to the swapchain
+	// present function is undefined as it only accepts unsigned integers as a parameter, so let's
```

Comment:
I think the `Present()` documentation suggests it only accepts 0-4.

Otherwise it seems fine (personally I would have overridden SetVSync to change the m_vsync value).